### PR TITLE
add hardware MR.DIY

### PR DESCRIPTION
**Pull request by MASangys**

### CODE REVIEW

File: brands/shop/hardware.json [hunk 1]
```diff
@@ -130,4 +130,17 @@
       "shop": "hardware"
     }
   }
-}
\ No newline at end of file
+  "shop/hardware|Home Hardware": {
```
Comment (matkoniecz) [inline brands/shop/hardware.json:133]: It probably should be "shop/hardware|MR.DIY"

File: brands/shop/hardware.json [hunk 2]
```diff
@@ -130,4 +130,17 @@
...
+    "countryCodes": ["my"],
+    "nomatch": [
+      "shop/doityourself|MR.DIY"
```
Comment (matkoniecz) [inline brands/shop/hardware.json:136]: Is "shop/doityourself|MR.DIY" added already to NSI if it is also a valid value?

Comment (matkoniecz) [reply brands/shop/hardware.json:136]: And if `shop=doityourself` should be replaced by `shop=hardware` then nomatch should be replaced by match

**Comment by MASangys** [comment]
I am pretty not sure about this. The category of this store should be home retailling. So best match it is hardware.
https://www.mrdiy.com/page/236/About-Us/

**Comment by matkoniecz** [comment]
Based on further discussion on telegram it is rather diy shop and I was asked to complete this PR.

I will do it now.

**Pull request closed without merging**